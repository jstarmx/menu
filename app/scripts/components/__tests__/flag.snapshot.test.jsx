import React from 'react';
import renderer from 'react-test-renderer';

import Flag from '../flag';

it('renders correctly', () => {
  const tree = renderer.create(
    <Flag code="en" />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
