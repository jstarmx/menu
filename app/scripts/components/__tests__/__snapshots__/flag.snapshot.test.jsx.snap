// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  className="flag"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<use xlink:href=\\"#en\\"></use>",
    }
  }
/>
`;
